style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
Kind change this to an Id, since it is only used once.

Reply:
- [ ] Awesome! but please can you clarify further what you meant by "id"? Perhaps, you meant "id selector". It's always informative when we give clear and easy to understand comments. **Note: I guess you wanted to write `Kindly` and not `Kind`, if so, please make necessary adjustment.**